server.go

```diff
@@ -178,17 +178,31 @@ func (s *server) UpdatePin(ctx context.Context, request v0.UpdatePinRequestObjec
 	})
 	if index == -1 {
 		log.Println("creating auth user", "accountID", *user.AccountUserID, "name", c.Name)
+		body := &models.SmartlockAuthCreate{
+			AccountUserID:       *user.AccountUserID,
+			SmartActionsEnabled: false,
+			Name:                ptr(c.Name),
+			Type:                13,
+			Code:                int32(request.Body.Code),
+			AllowedWeekDays:     ptr(int32(s.cfg.Nuki.AllowedWeekDays)),
+		}
+	if s.cfg.Nuki.AllowedFromDate != nil && s.cfg.Nuki.AllowedUntilDate != nil{
```

Comment:
there's a way to say 'unlimted' but not sure how.

Reply:
leave it empty?